docs/supported_ops.rst

```diff
@@ -1,73 +1,16 @@
-Supported operations
-====================
+Operations
+==========
 
-Here is a list of the supported operations:
+.. currentmodule:: nvidia.dali
 
-- | **CPU** operator means that the operator can be scheduled on the CPU.
-  | The outputs of CPU operators may be used as regular inputs and to provide per-sample parameters
-    for other operators through tensor arguments.
-- **GPU** operator means that the operator can be scheduled on the GPU. Their outputs can only be
-  used as
-  regular inputs for other GPU operators and pipeline outputs.
-- **Mixed** operator means that the operator accepts CPU inputs and produces GPU outputs.
-- **Sequences** means that the operator can produce or accept as an input a sequence
-  (for example, a video).
-- **Volumetric** means that the operator supports 3D data processing.
+DALI's operations or "functions" are contained in the ``dali.fn`` module and their names are snake-cased
+(for example ``dali.fn.image_decoder``).
```

Comment:
I don't think we need that here.

Reply:
The example or the whole sentence?

Reply:
The whole sentence. Like we don't describe the naming convention in the legacy page. It has only value (IMHO) when you compare the legacy with the fn API. In this page we don't have to compare these two approaches.